localisation/00_zuwc_united_localisation.csv

```diff
@@ -2411,7 +2412,7 @@ ERA_CHAR_INFO_60003;Crown Prince Arthas Menethil of Lordaeron is the son of King
-ERA_CHAR_INFO_60001;King Terenas Menethil II of Lordaeron, was born in the court of Loradaeron as a member of the royal family, which had ruled the Kingdom for generations. He grew up to become a wise and just ruler, beloved by his people for bringing prosperity and peace for most of his long reign. He is an old friend of Thoras Trollbane, the King of Stromgarde and that could play a crucial role in the future. However, with rumours of a new threat coming from beyond Azeroth, as the ruler of the largest human kingdom Terenas must prepare his banners for war as the very fate of this world could very well be in his hands.;;K�nig Terenas Menethil II. von Lordaeron wurde am Hof von Loradaeron als Mitglied der k�niglichen Familie geboren, die das K�nigreich seit Generationen regiert hatte. Er wuchs zu einem weisen und gerechten Herrscher heran, der von seinem Volk geliebt wird, weil er f�r die meiste Zeit seiner langen Herrschaft Wohlstand und Frieden gebracht hat. Er ist ein alter Freund von Thoras Trollbane, dem K�nig von Stromgarde, und dies k�nnte in Zukunft eine entscheidende Rolle spielen. Doch nun, da Ger�chte �ber eine neue Bedrohung von jenseits von Azeroth kommen, muss Terenas, der Herrscher des gr��ten menschlichen K�nigreichs, seine Banner f�r den Krieg vorbereiten, denn das Schicksal dieser Welt k�nnte sehr wohl in seinen H�nden liegen.;;;;;;;;;;;x
+ERA_CHAR_INFO_60001;King Terenas Menethil II of Lordaeron, was born in the court of Loradaeron as a member of the royal family, which had ruled the Kingdom for generations. He grew up to become a wise and just ruler, beloved by his people for bringing prosperity and peace for most of his long reign. He is an old friend of Thoras Trollbane, the King of Stromgarde and that could play a crucial role in the future. However, with rumours of a new threat coming from beyond Azeroth, as the ruler of the largest human kingdom Terenas must prepare his banners for war as the very fate of this world could very well be in his hands.;;K�nig Terenas Menethil II. von Lordaeron wurde am Hof von Lordaeron als Mitglied der k�niglichen Familie geboren, die das K�nigreich seit Generationen regiert hatte. Er wuchs zu einem weisen und gerechten Herrscher heran, der von seinem Volk geliebt wird, weil er f�r die meiste Zeit seiner langen Herrschaft Wohlstand und Frieden gebracht hat. Er ist ein alter Freund von Thoras Trollbane, dem K�nig von Stromgarde, und dies k�nnte in Zukunft eine entscheidende Rolle spielen. Doch nun, da Ger�chte �ber eine neue Bedrohung von jenseits von Azeroth kommen, muss Terenas, der Herrscher des gr��ten menschlichen K�nigreichs, seine Banner f�r den Krieg vorbereiten, denn das Schicksal dieser Welt k�nnte sehr wohl in seinen H�nden liegen.;;;;;;;;;;;x
```

Comment:
```suggestion
ERA_CHAR_INFO_60001;King Terenas Menethil II of Lordaeron, was born in the court of Loradaeron as a member of the Menethil Dynasty, which had ruled the Kingdom for generations. Beloved by his people for his wise and just rule, bringing peace and prosperity during his long reign. He is an old friend of Thoras Trollbane, King of Stromgarde and a crucial figure in events to come. However, with rumours of a new threat coming from beyond Azeroth and as the ruler of the largest human kingdom, Terenas must prepare his banners for war as the very fate of this world could very well be in his hands.;;K�nig Terenas Menethil II. von Lordaeron wurde am Hof von Lordaeron als Mitglied der k�niglichen Familie geboren, die das K�nigreich seit Generationen regiert hatte. Er wuchs zu einem weisen und gerechten Herrscher heran, der von seinem Volk geliebt wird, weil er f�r die meiste Zeit seiner langen Herrschaft Wohlstand und Frieden gebracht hat. Er ist ein alter Freund von Thoras Trollbane, dem K�nig von Stromgarde, und dies k�nnte in Zukunft eine entscheidende Rolle spielen. Doch nun, da Ger�chte �ber eine neue Bedrohung von jenseits von Azeroth kommen, muss Terenas, der Herrscher des gr��ten menschlichen K�nigreichs, seine Banner f�r den Krieg vorbereiten, denn das Schicksal dieser Welt k�nnte sehr wohl in seinen H�nden liegen.;;;;;;;;;;;x
```

Reply:
modified, pls check again. I did not change the part about Thoras, because we don't want to spoil things that are yet to happen.

Reply:
dealt with in discord